src/app/icd/server/ICDConfigurationData.h

```diff
@@ -61,6 +61,8 @@ class ICDConfigurationData
 
     System::Clock::Milliseconds16 GetActiveModeThreshold() { return mActiveThreshold; }
 
+    System::Clock::Milliseconds32 GetMaxStayActiveDuration() { return kMaxGuaranteedStayActiveDuration; }
```

Comment:
For clarity, i would the `max` bits since it is just the GaranteedStayActiveDuration.